vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/Upload.java

```diff
@@ -96,18 +106,16 @@ public UploadIcon() {
      * The receiver must be set before performing an upload.
      */
     public Upload() {
-        // Get a server round trip for upload error and success.
-        addUploadErrorListener(event -> {
-        });
-
-        addUploadSuccessListener(event -> {
-        });
```

Comment:
These empty listeners seem to be not needed anymore after adding `upload-success` and `upload-error` listeners to handle upload finished event - see https://github.com/vaadin/vaadin-upload-flow/pull/144 and https://github.com/vaadin/vaadin-upload-flow/pull/152